backend/main.py

```
@@ -0,0 +1,31 @@
+from fastapi import FastAPI, HTTPException
```

Ruff (0.11.9)

`fastapi.HTTPException` imported but unused

Remove unused import: `fastapi.HTTPException`

(F401)